src/app/components/map/banners/banner-dialog/banner-dialog.component.scss

```diff
@@ -1,17 +1,40 @@
-@import "asf-theme";
+@import url("asf-theme");
```

Comment:
I think we ran into an issue with code-factor automatically adding this in and breaking builds?